pom.xml

```diff
@@ -420,6 +422,20 @@
             <version>${org.mockito.mockito-all-version}</version>
         </dependency>
 
+            <dependency>
+                <groupId>org.powermock</groupId>
```

Comment:
If we use only Mockito, then we can not test final class, static method and we have other constraints. So in this case I additionally used PowerMock, that doesn't have this restrictions